openshift-install-powervs

```diff
@@ -1102,6 +1102,12 @@ function variables {
 	  error "IBM Cloud VPC subnet name must be non-empty for using IBM Cloud Services"
 	fi
 	echo "use_ibm_cloud_services = \"true\"" >> $VAR_TEMPLATE
+	question "Enter IBM Cloud VPC Infrastructure region"
```

Comment:
Enter IBM Cloud VPC Infrastructure region. (Power VS region will be used for VPC if you do not provide any value).

Reply:
@yussufsh updated the change.